src/snowflake/cli/api/project/schemas/updatable_model.py

```diff
@@ -131,7 +131,10 @@ def __init_subclass__(cls, **kwargs):
 
         # Add Pydantic validation wrapper around all fields except `DiscriminatorField`s
         for field_name in field_annotations:
-            if not cls._is_entity_type_field(field_values.get(field_name)):
+            field = field_values.get(field_name)
+            if field is None:
```

Comment:
Not sure if this is accurate. what if we have a class:
```
class A: {
 B: int
}
```
Wouldn't B value be None in that case?

Reply:
(Alternatively, not sure if there is a way to check if Field type is PrivateField, or if it starts with _)

Reply:
Just found out, you can do the same as checking FieldInfo:
`from pydantic.fields import FieldInfo, ModelPrivateAttr`

Then you can check if type is `ModelPrivateAttr`